### PR TITLE
DOC-5031: Added missing parameter and more info about schemas

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

File: content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md [hunk 1]
```diff
@@ -214,22 +214,33 @@ To create the schema manually, do the following:
...
    {{% tab name="macOS" %}}
    ```sh
-   corda-cli.sh database spec -c -l /tmp/db
+   corda-cli.sh database spec -g config:CONFIG,rbac:RBAC,crypto:CRYPTO -c -l /tmp/db
    ```
    {{% /tab %}}
    {{% tab name="Windows" %}}
    ```shell
-   corda-cli.cmd database spec -c -l /tmp/db
+   corda-cli.sh database spec -g config:CONFIG,rbac:RBAC,crypto:CRYPTO -c -l /tmp/db
```
Comment (ShYogesh-R3) [inline content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md:243]: this should be "corda-cli.cmd" for windows.

**Review by ShYogesh-R3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md (hunk 1 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md]: @ShYogesh-R3 updated

Comment (yarivus) [reply content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md]: that's right

**Review by yarivus — CHANGES_REQUESTED** [review]

**Review by yarivus — APPROVED** [review]
LGTM

**Review by ShYogesh-R3 — DISMISSED** [review]

File: content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md [hunk 2]
```diff
@@ -214,22 +214,33 @@ To create the schema manually, do the following:
...
+2. Use the Corda CLI to generate DML files for creating the database schema to use for each of the CRYPTO, CONFIG, and RBAC tables.
+If the tables are not specified, then they will be created in the default schema and the next steps in this procedure will need updating to reflect this.
+
+{{< note >}}
+
+Corda 5 no longer forces users to use any schema. You can choose your own schema names, or even use the public schema
+(most useful where there are multiple databases in use, for example, one for RBAC, one for CRYPTO, and so on).
+If you do want to use specific schemas, you have to provide those or the tool assumes that you want to use the public schema.
+
+{{</ note >}}
```
Comment (davidcurrie) [inline content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md:226]: I wouldn't include this note - I think the paragraph before says everything that they need to know.

File: content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md [hunk 3]
```diff
@@ -214,22 +214,33 @@ To create the schema manually, do the following:
...
+
+The following command generates the files in the directory `/tmp/db`:
```
Comment (davidcurrie) [inline content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md:228]:
```suggestion
The following command specifies that the `CONFIG`, `RBAC` and `CRYPTO` schema should be used for the corresponding components and generates the files in the directory `/tmp/db`:
```

**Review by davidcurrie — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md (hunk 2 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md]: removed

**Review by paulmoloneyr3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thx @AnnaK-R3

**Pull request merged**